src/Adapter/Cart/QueryHandler/GetCartForOrderCreationHandler.php

```diff
@@ -553,6 +553,8 @@ private function buildCartProduct(
             \Tools::ps_round($product['total'], $currency->precision),
             $this->contextLink->getImageLink($product['link_rewrite'], $product['id_image'], 'small_default'),
             $this->getProductCustomizedData($cart, $product),
+            Product::getQuantity((int) $product['id_product']),
+            (bool) Product::isAvailableWhenOutOfStock($product['out_of_stock']),
```

Comment:
The funny thing, this method returns an int :exploding_head:

Reply:
Fixed